contentcuration/contentcuration/viewsets/base.py

```diff
@@ -301,6 +306,40 @@ def get_serializer_class(self):
         # Hack to prevent the renderer logic from breaking completely.
         return Serializer
 
+    def get_edit_queryset(self):
```

Comment:
Should these methods be returning valid results on a read only viewset? Is the idea that the `ReadOnlyValuesViewset` is serving as the base class for `ValuesViewset`?

Reply:
Yeah - it was simpler to define most of the methods on the `ReadOnlyValuesViewset` and just implement the additional methods that make the DRF router add the additional methods for create, update, and delete operations on the main viewset.

Reply:
Also just realized I forgot to add delete....

Reply:
Okay, cool, just wanted to check! 👍

Reply:
Oh.. I did implement delete, I just used the DRF mixin for it... disregard!